iOS/Flows/Circles/CircleCoordinator.swift

```diff
@@ -9,16 +9,89 @@
 import Foundation
 
 class CircleCoordinator: PresentableCoordinator<Void> {
-
-    lazy var circleVC = CircleViewController()
-
+    
+    let circleVC: CircleViewController
+    
+    init(with cirlce: Circle,
+         router: Router,
+         deepLink: DeepLinkable?) {
+        
+        self.circleVC = CircleViewController(with: cirlce)
+        super.init(router: router, deepLink: deepLink)
+    }
+    
     override func toPresentable() -> PresentableCoordinator<Void>.DismissableVC {
         return self.circleVC
     }
-
+    
     override func start() {
         super.start()
-
-       
+        
+        self.circleVC.button.didSelect { [unowned self] in
+            self.presentCircleTitleAlert()
+        }
+        
+        self.circleVC.$selectedItems.mainSink { [unowned self] items in
+            guard let itemType = items.first else { return }
+            switch itemType {
+            case .item(let circleItem):
+                if circleItem.canAdd {
+                    self.presentPeoplePicker()
+                }
+            }
+        }.store(in: &self.cancellables)
+    }
+    
+    func presentPeoplePicker() {
+        
+        self.removeChild()
+        let coordinator = PeopleCoordinator(router: self.router, deepLink: self.deepLink)
+        
+        self.addChildAndStart(coordinator) { [unowned self] people in
```

Comment:
Update to how the people coordinator works. It returns a person array, not connections. This way we can have both users/contacts.